src/systems/ec/implementations/react/templates/pages/person/src/PersonPage.jsx

```diff
@@ -0,0 +1,907 @@
+    ))}
+  </BreadcrumbStandardised>
+);
+const breadcrumbHarmonised1 = (
+  <BreadcrumbHarmonised
+    {...breadcrumbContent}
+    className="ecl-breadcrumb-harmonised--group1"
+    data-ecl-auto-init="BreadcrumbHarmonised"
+  >
+    {breadcrumbItems.map(item => (
+      <BreadcrumbHarmonisedItem {...item} key={item.label} />
+    ))}
+  </BreadcrumbHarmonised>
+);
+const breadcrumbHarmonised2 = (
+  <BreadcrumbHarmonised
+    {...breadcrumbContent}
+    className="ecl-breadcrumb-harmonised--group2"
+    data-ecl-auto-init="BreadcrumbHarmonised"
+  >
+    {breadcrumbItems.map(item => (
+      <BreadcrumbHarmonisedItem {...item} key={item.label} />
+    ))}
+  </BreadcrumbHarmonised>
+);
+
+const pageHeaderContent = {
+  meta: 'Commissioner (2014 - 2019)',
+  title: 'Vytenis Andriukaitis',
+  description: 'Health & Food Safety',
+};
+
+const PersonPage = ({ siteHeader, footer, template }) => (
+  <Fragment>
+    {template === 'core' && (
+      <SiteHeaderCore {...siteHeader} data-ecl-auto-init="SiteHeaderCore" />
+    )}
+    {template === 'standardised' && (
+      <SiteHeaderStandardised
+        {...siteHeader}
+        data-ecl-auto-init="SiteHeaderStandardised"
+      />
+    )}
+    {template === 'harmonised-g1' && (
+      <SiteHeaderHarmonised
+        {...siteHeader}
+        data-ecl-auto-init="SiteHeaderHarmonised"
+        className="ecl-site-header-harmonised--group1"
+      />
+    )}
+    {template === 'harmonised-g2' && (
+      <SiteHeaderHarmonised
+        {...siteHeader}
+        data-ecl-auto-init="SiteHeaderHarmonised"
+        className="ecl-site-header-harmonised--group2"
+      />
+    )}
+
+    {template === 'core' && (
+      <PageHeaderCore {...pageHeaderContent} breadcrumb={breadcrumbCore} />
+    )}
+    {template === 'standardised' && (
+      <PageHeaderStandardised
+        {...pageHeaderContent}
+        breadcrumb={breadcrumbStandardised}
+      />
+    )}
+    {template === 'harmonised-g1' && (
+      <PageHeaderHarmonised
+        {...pageHeaderContent}
+        breadcrumb={breadcrumbHarmonised1}
+        className="ecl-page-header-harmonised--group1"
+      />
+    )}
+    {template === 'harmonised-g2' && (
+      <PageHeaderHarmonised
+        {...pageHeaderContent}
+        breadcrumb={breadcrumbHarmonised2}
+        className="ecl-page-header-harmonised--group2"
+      />
+    )}
+
+    <div className="ecl-u-pv-xl ecl-u-pv-lg-3xl">
+      <div className="ecl-container ecl-u-type-m">
+        <div className="ecl-row" data-ecl-inpage-navigation-container>
+          <div className="ecl-col-12 ecl-col-lg-3">
+            <img
+              className="ecl-u-width-100 ecl-u-d-none ecl-u-d-md-block ecl-u-mb-xl ecl-u-mb-lg-3xl"
+              alt="example"
+              src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/27/Emojione_1F464.svg/1024px-Emojione_1F464.svg.png"
+            />
+            <aside className="ecl-u-height-100">
+              <InpageNavigation
```

Comment:
InpageNavigation menu is very long.. Therefore when we scroll and we reach the end of the page, there is an overlap with the footer. This seems to be more of a component problem.

Reply:
You're right, it's a more global issue. I put `ecl-u-z-navigation` which is 10, but it's not good idea to add more than this, because if it, then we'll see other issues like inpage nav being seen when language switcher is open (modal), etc.

Reply:
@kalinchernev if we remove the person image above the inpageNavigation then it work perfectly. I think having the image above is not a good idea in terms of design. I think we should report this to designer. I see also a duplicate image in the content for responsiveness.